OSF Preprints.js

```diff
@@ -64,84 +64,58 @@ function getSearchResults(doc, checkOnly) {
 	return found ? items : false;
 }
 
-function doWeb(doc, url) {
+async function doWeb(doc, url) {
 	if (detectWeb(doc, url) == "multiple") {
-		Zotero.selectItems(getSearchResults(doc, false), function (items) {
-			if (items) ZU.doGet(constructAPIURL(Object.keys(items)), osfAPIImport);
+		// only work for preprint type like previous code
+		Zotero.selectItems(getSearchResults(doc, false), async function (items) {
+			if (items) {
+				for (let url of Object.keys(items)) {
+					await scrape(doc, url);
```

Comment:
I think the multiple results issue comes from `doc` parameter here. It should be the document of the corresponding URL based on current scrap logic. But I failed to fix it